tools/filter_cells_and_genes/filter_cells_and_genes.xml

```diff
@@ -0,0 +1,270 @@
+         <param name="package" value="Seurat"/>
+         <param name="min_genes" value="10"/>
+         <param name="min_cells" value="3"/>
+         <output name="output_plot" file="counts_Seurat_filter_plots.pdf" ftype="pdf"/>
+         <output name="output_matrix" file="counts_Seurat_filtered.tab" ftype="tabular"/>
+    </test>
+    <test>
+         <param name="file" value="counts_with_mito.tab" ftype="tabular"/>
+         <param name="file_sep" value="tab"/>
+         <param name="file_header" value="TRUE"/>
+         <param name="is_mito" value="TRUE"/>
+         <param name="header_mito" value="MT-"/>
+         <param name="package" value="scater"/>
+         <param name="gene_filter" value="low.abundances"/>
+         <param name="min_cells" value="1"/>
+         <output name="output_plot" file="counts_with_mito_scater_low.abundances_filter_plots.pdf" ftype="pdf"/>
+         <output name="output_matrix" file="counts_with_mito_scater_low.abundances_filtered.tab" ftype="tabular"/>
+    </test>
+    <test>
+         <param name="file" value="counts_with_mito.tab" ftype="tabular"/>
+         <param name="file_sep" value="tab"/>
+         <param name="file_header" value="TRUE"/>
+         <param name="is_mito" value="TRUE"/>
+         <param name="header_mito" value="MT-"/>
+         <param name="package" value="scater"/>
+         <param name="gene_filter" value="min.cells"/>
+         <param name="min_cells" value="3"/>
+         <output name="output_plot" file="counts_with_mito_scater_min.cells_filter_plots.pdf" ftype="pdf"/>
+         <output name="output_matrix" file="counts_with_mito_scater_min.cells_filtered.tab" ftype="tabular"/>
+    </test>
+    <test>
+         <param name="file" value="counts_with_mito.tab" ftype="tabular"/>
+         <param name="file_sep" value="tab"/>
+         <param name="file_header" value="TRUE"/>
+         <param name="is_mito" value="TRUE"/>
+         <param name="header_mito" value="MT-"/>
+         <param name="package" value="Seurat"/>
+         <param name="min_genes" value="10"/>
+         <param name="min_cells" value="3"/>
+         <output name="output_plot" file="counts_with_mito_Seurat_filter_plots.pdf" ftype="pdf"/>
+         <output name="output_matrix" file="counts_with_mito_Seurat_filtered.tab" ftype="tabular"/>
+    </test> 
+   </tests>
+   <help>
+
+**What it does**
+
+Filter_cells_and_genes takes a tabular table of raw read counts, each column corresponding to a cell and each row corresponding to a
+gene, and returns (1) a PDF with quality controls plots (like the distribution of library sizes or expressed genes by cell, frequency of 
+expression or the most expressed genes) and (2) a filtered table of counts.
+
+You can filter out low quality cells from your raw expression table :
+
+- Based on expression level of mitochondrial genes :
+
+    - Using R package scater: remove cells that detect mitochondrial genes 3 times higher than the median of all cells
```

Comment:
Be careful with the term "mitochondrial genes" it may be misleading. I understand that you speak of genes that are encoded by the mitochondrial genome. But some others could understand that this include gene encoding mitochondrial protein, which is not the case here, right ?
"mitochondrial genes (i.e. genes mapped to the mitochondrial chromosome)" would be more explicit

Reply:
This is how I understand from researches I made, it's appear that it corresponds to genes mapped to the chrM. In all tools I used, they use the term of "mitochondrial genes"